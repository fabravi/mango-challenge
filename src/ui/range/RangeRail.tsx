import styles from "./rangerail.module.scss";

export default function RangeRail() {
  return <span className={styles.rail}></span>;
}
