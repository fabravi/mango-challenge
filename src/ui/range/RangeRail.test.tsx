import { render } from "@testing-library/react";
import RangeRail from "./RangeRail";

describe("RangeRail", () => {
  test("should render correctly", () => {
    const { container } = render(<RangeRail />);
    expect(container).toMatchSnapshot();
  });

  test("should render the rail", () => {
    const { container } = render(<RangeRail />);
    const rail = container.querySelector("span");
    expect(rail).toHaveClass("rail");
  });
});
