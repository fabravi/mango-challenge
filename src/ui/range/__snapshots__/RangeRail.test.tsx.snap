// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeRail should render correctly 1`] = `
<div>
  <span
    class="rail"
  />
</div>
`;
